vscode_extension/package.json

```diff
@@ -98,6 +99,12 @@
         "category": "Sorbet",
         "enablement": "workbenchState != empty"
       },
+      {
+        "command": "sorbet.configureHighlightUntyped",
+        "title": "Configure untyped code highlighting",
+        "category": "Sorbet",
+        "enablement": "workbenchState != empty"
+      },
```

Comment:
Alpha-sorting applies particularly to this section since  different `enablement` rules cause funny issue (not from experience).